includes/notes/class-wc-stripe-upe-stripelink-note.php

```diff
@@ -0,0 +1,100 @@
+<?php
+/**
+ * Display a notice to merchants to inform about UPE.
+ *
+ * @package WooCommerce\Payments\Admin
+ */
+
+use Automattic\WooCommerce\Admin\Notes\NoteTraits;
+use Automattic\WooCommerce\Admin\Notes\Note;
+use Automattic\WooCommerce\Admin\Notes\WC_Admin_Note;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class WC_Stripe_UPE_StripeLink_Note
+ */
+class WC_Stripe_UPE_StripeLink_Note {
+	use NoteTraits;
+
+	/**
+	 * Name of the note for use in the database.
+	 */
+	const NOTE_NAME = 'wc-stripe-upe-stripelink-note';
+
+	/**
+	 * Link to enable the UPE in store.
+	 */
+	const NOTE_DOCUMENTATION_URL = 'https://woocommerce.com/document/stripe/#section-11';
+
+	/**
+	 * Get the note.
+	 */
+	public static function get_note() {
+		$note_class = self::get_note_class();
+		$note       = new $note_class();
+
+		$note->set_title( __( 'Increase conversion at checkout', 'woocommerce-gateway-stripe' ) );
+		$note->set_content( __( 'Reduce cart abandonment and create a frictionless checkout experience with Link by Stripe. Link autofills your customer’s payment and shipping details so they can check out in just six seconds with the Link optimized experience. That’s 9x faster than shoppers who don’t use Link. Link increases conversion rates by over 7% for logged-in Link customers.', 'woocommerce-gateway-stripe' ) );
```

Comment:
It looks like the note content is trimmed down after "over...", and there's no way to expand it. I'm guessing this is a limitation of the WC Admin inbox, and it's only supposed to look like that if any of the buttons link to the continuation of the note 🤔.

Should we ask the author of the note copy and maybe shorten it a little bit?

<img width="702" alt="WooCommerce Stripe Link inbox notification" src="https://user-images.githubusercontent.com/5509901/197618614-d910e883-5ea3-45b4-af04-b047e143449b.png">

Reply:
changed in [347b35f](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2462/commits/347b35fb0c1bd22ee23204ffadb70b41961723aa). thx